static/app/components/replays/replayController.tsx

```diff
@@ -60,27 +62,26 @@ function ReplayCurrentTime() {
 
 function ReplayPlaybackSpeed({speedOptions}: {speedOptions: number[]}) {
   const {setSpeed, speed} = useReplayContext();
-
   return (
-    <ButtonBar active={String(speed)} merged>
-      {speedOptions.map(opt => (
-        <Button
-          key={opt}
-          size="xsmall"
-          barId={String(opt)}
-          onClick={() => setSpeed(opt)}
-          title={t('Set playback speed to %s', `${opt}x`)}
-        >
-          {opt}x
-        </Button>
-      ))}
-    </ButtonBar>
+    <StyledOptionSelector
+      title={`${t('Speed')}`}
+      selected={`${speed}`}
+      options={speedOptions.map(speedOption => ({
+        value: `${speedOption}`,
```

Comment:
The `speedOption` is a number, feels weird to convert to a string

```suggestion
        value: speedOption,
```

I hope this makes the `onChange` handler understand that it's a number type too, then we can drop the parseInt call and be more typesafe.